adserver/admin.py

```diff
@@ -719,6 +722,25 @@ def has_add_permission(self, request):
         """Clicks and views cannot be added through the admin."""
         return False
 
+    def refund_impressions(self, request, queryset):
```

Comment:
I worry a little bit about not having a confirmation step here, but it should be fine if we accidentally hit it for 1 view or something. But likely need to be a bit more defensive here in the future.

Reply:
Is there a good way to add a confirmation to the admin?

Reply:
There's this: https://docs.djangoproject.com/en/2.2/ref/contrib/admin/actions/#actions-that-provide-intermediate-pages

I'll see if I can add one quickly.